stable/weave-scope/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: weave-scope
-version: 1.1.5
+version: 1.1.6
 appVersion: 1.11.5
 description: A Helm chart for the Weave Scope cluster visualizer.
```

Comment:
Idea (not for this PR): we might want to start specifying `kubeVersion` at some point, especially when we drop support for old K8s.

Reply:
I would like to do it when ensuure the minimum version number.